logic/transaction_pool.js

```diff
@@ -787,14 +789,50 @@ __private.isTransactionInUnconfirmedQueue = function(transaction) {
 	return typeof self.unconfirmed.index[transaction.id] === 'number';
 };
 
+/**
+ * Check if a transaction is sent from multisignature account.
+ *
+ * @private
+ * @param {Object} transaction - Transaction object
+ * @param {function} cb - Callback
+ * @returns {Boolean} True if the transaction is type 4 or sent from a multisig account
+ */
+__private.isMultisignatureTransaction = function(transaction, cb) {
+	async.waterfall(
+		[
+			function getAccount(waterCb) {
+				if (transaction.senderPublicKey) {
+					modules.accounts.getAccount(
+						{ publicKey: transaction.senderPublicKey },
+						waterCb
+					);
```

Comment:
The definition says: 
> sent from a multisig account

But the logic here is just checking for a normal account, not the multisig account. If you really need a multisig account you can use `logic.account. getMultiSignature`